src/neuroconv/datainterfaces/ecephys/phy/phydatainterface.py

```diff
@@ -12,7 +12,7 @@ class PhySortingInterface(BaseSortingExtractorInterface):
     def __init__(
         self,
         folder_path: FolderPathType,
-        exclude_cluster_groups: Optional[list] = None,
+        exclude_cluster_groups: Optional[list[str]] = None,
```

Comment:
Doesn't this have to be a `List` from `typing`?

Reply:
I guess they both produce similar `GenericAlias` these days - IDK if I expect `get_schema_from_method_signature` to work on this, however - I don't think it has logic for recursing nested levels of annotation besides `Union`/`Optional` (https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/utils/json_schema.py#L51)